packages/compass/src/app/components/home.tsx

```diff
@@ -318,6 +362,11 @@ function Home({
                 getAutoConnectInfo={
                   hasDisconnectedAtLeastOnce ? undefined : getAutoConnectInfo
                 }
+                useConnectionImportExportModalRenderer={
```

Comment:
It's a bit unconventional to pass the whole hook as a component prop, I'd suggest to split it to actual props that you kinda want here:

```suggestion
                onOpenImportExportConnections={...}
                renderImportExportConnectionsModals={...}
```

Or if it's possible to not pass connections list to the modals, I'd suggest to do the same thing that I suggested for LegacyConnectionModal and just render it right here instead of through the CompassConnections. Then the only thing you need is the `onOpen` callback prop, which would make the interface of this component to be a very normal React component